execution/executor-benchmark/src/lib.rs

```diff
@@ -174,8 +195,19 @@ pub fn run_benchmark<V>(
             "raw transfer".to_string()
         }
     );
-    info!("Overall TPS: {} txn/s", delta_v as f32 / elapsed);
-    info!("Overall GPS: {} gas/s", delta_gas as f32 / elapsed);
+    info!("Overall TPS: {} txn/s", delta_v as f64 / elapsed);
+    info!("Overall GPS: {} gas/s", delta_gas as f64 / elapsed);
+
+    let time_in_execution = APTOS_EXECUTOR_EXECUTE_BLOCK_SECONDS.get_sample_sum() - start_execution_total;
+    info!("Overall fraction of total: {:.3} in execution", time_in_execution / elapsed);
+    info!("Overall fraction of execution {:.3} in VM", (APTOS_EXECUTOR_VM_EXECUTE_BLOCK_SECONDS.get_sample_sum() - start_vm_only) / time_in_execution);
+    for (prefix, top_level,  other_label) in other_labels {
+        let fraction_in_label = (APTOS_EXECUTOR_OTHER_TIMERS_SECONDS.with_label_values(&[other_label]).get_sample_sum() - start_by_other.get(other_label).unwrap()) / time_in_execution;
+        if top_level || fraction_in_label > 0.01 {
+            info!("Overall fraction of execution {:.3} in {} {}", fraction_in_label, prefix, other_label);
```

Comment:
nit: I find TPS metrics more intuitive than execution time. Thoughts on output TPS per state in addition to execution time or may be just the TPS?

Reply:
@sitalkedia  - added